doc/users/fonts.rst

```diff
@@ -0,0 +1,120 @@
+   :header-rows: 1
+
+   * - Type 1 (PDF)
+     - Type 3 (PDF/PS)
+     - TrueType (PDF)
+   * - One of the oldest types, introduced by Adobe
+     - Similar to Type 1 in terms of introduction
+     - Newer than previous types, used commonly today, introduced by Apple
+   * - Restricted subset of PostScript, charstrings are in bytecode
+     - Full PostScript language, allows embedding arbitrary code
+       (in theory, even render fractals when rasterizing!)
+     - Include a virtual machine that can execute code!
+   * - These fonts support font hinting
+     - Do not support font hinting
+     - Hinting supported (virtual machine processes the "hints")
+   * - Non-subsetted through Matplotlib
+     - Subsetted via external module `ttconv <https://github.com/sandflow/ttconv>`_
+     - Subsetted via external module `fonttools <https://github.com/fonttools/fonttools>`_
+
+NOTE: Adobe will disable support for authoring with Type 1 fonts in
+January 2023. `Read more here. <https://helpx.adobe.com/fonts/kb/postscript-type-1-fonts-end-of-support.html>`_
+
+Special Mentions
+~~~~~~~~~~~~~~~~
+- Type 42 fonts (PS):
+
+  - PostScript wrapper around TrueType fonts
+  - 42 is the `Answer to Life, the Universe, and Everything! <https://en.wikipedia.org/wiki/Answer_to_Life,_the_Universe,_and_Everything>`_
+  - Matplotlib uses an external library called `fonttools <https://github.com/fonttools/fonttools>`_
+    to subset these types of fonts
+
+- OpenType fonts:
+
+  - OpenType is a new standard for digital type fonts, developed jointly by
+    Adobe and Microsoft
+  - Generally contain a much larger character set!
+  - Limited Support with Matplotlib
+
+
+Subsetting
+----------
+Matplotlib is able to generate documents in multiple different formats. Some of
+those formats (for example, PDF, PS/EPS, SVG) allow embedding font data in such
+a way that when these documents are visually scaled, the text does not appear
+pixelated.
+
+This can be achieved by virtually embedding the *whole* font file within the
+output document. However, this can lead to **very large documents**, wherein
+most of the size bandwidth is captured by that font file data.
+
+Font Subsetting is a way to embed only the *required* glyphs within the
+documents. Fonts can be considered as a collection of glyphs, so ultimately the
+goal is to find out *which* glyphs are required for a certain array of
+characters, and embed only those within the output.
+
+.. note::
+  The role of subsetter really shines when we encounter characters like **ä**
+  (composed by calling subprograms for **a** and **¨**); since the subsetter
+  has to find out *all* such subprograms being called by every glyph included
+  in the subset, this is a generally difficult problem!
+
+Luckily, Matplotlib uses a fork of an external dependency called
+`ttconv <https://github.com/sandflow/ttconv>`_, which helps in embedding and
+subsetting stuff. (however, recent versions have moved away from ttconv to pure
```

Comment:
```suggestion
subsetting glyphs. (however, recent versions have moved away from ttconv to pure
```

Reply:
Changed it to 'font data', since all this isn't just for the glyphs but also for other metadata